### PR TITLE
unique sql constraint on sale_channel_magento_party

### PR DESCRIPTION
**Pull request by cerebro1**

### CODE REVIEW

**Comment by cerebro1** [comment]
https://3.basecamp.com/3194017/buckets/421793/todos/112272737

File: party.py [hunk 1]
```diff
@@ -171,8 +171,12 @@ def __setup__(cls):
         """
         super(MagentoWebsiteParty, cls).__setup__()
         cls._error_messages.update({
-            'party_exists': 'A party must be unique in a channel'
+            'unique_party_exists': 'A party must be unique in a channel'
```
Comment (riteshshrv) [inline party.py]: revert

Comment (tarunbhardwaj) [inline party.py]: "Party already exist for this channel."

File: party.py [hunk 2]
```diff
@@ -171,8 +171,12 @@ def __setup__(cls):
...
         })
+        cls._sql_constraints += [
+            ('unique_party', 'UNIQUE(magento_id, channel)',
+             'unique_party_exists')
+        ]
```
Comment (riteshshrv) [inline party.py]: Add a new error message and use it: `Party and Magento ID should be unique per channel`

File: party.py (hunk 1 above)
Comment (riteshshrv) [inline party.py]: revert; no change in this line except for a trailing comma

File: party.py [hunk 3]
```diff
@@ -171,8 +171,14 @@ def __setup__(cls):
         """
         super(MagentoWebsiteParty, cls).__setup__()
         cls._error_messages.update({
-            'party_exists': 'A party must be unique in a channel'
+            'party_exists': 'Party already exist for this channel.',
```
Comment (riteshshrv) [inline party.py]: make this mesage similar to what it was

**Pull request closed without merging**